src/bundle/ui-dev/src/modules/universal-discovery/components/selected-items/selected.items.panel.item.js

```diff
@@ -78,7 +81,12 @@ SelectedItemsPanelItem.propTypes = {
         resource: PropTypes.string.isRequired,
     }).isRequired,
     name: PropTypes.string.isRequired,
-    description: PropTypes.string.isRequired,
+    description: PropTypes.node.isRequired,
+    extraContent: PropTypes.oneOfType([PropTypes.arrayOf(PropTypes.node), PropTypes.node]),
```

Comment:
Is it necessary to allow here array? Wouldn't it be better to reduce possibilities to just node and require to wrap more than one item into some other JSX element?